### PR TITLE
Navbar  welcome page created

### PR DESCRIPTION
**Pull request by tinatsina**
created the mobile version of the portfolio website with
-navbar and icon at the right
-images inserted as absolutes
-mobile version of the website created

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    Start collaboration
+                </button>
+            </div>
+            <div class="sub-container-2"></div>
+        </div>
+    </section>
+    <!--- Box Image section -->
+    <div class="image-section">
+        <div class="sub-image"></div>
+    </div>
```
Comment (topeogunleye) [inline index.html:46]: - [ ] As required, please, I kindly suggest that you add the Projects section of your portfolio and the about me section. Kindly check your [design templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for guidance.

| Projects  Section | About me Section |
| ------------- | ------------- |
| Screenshot from 2022-05-30 18-59-35  | Screenshot from 2022-05-30 19-00-03 |
| Screenshot from 2022-05-30 18-59-51 | Content Cell  |

- [ ] Please kindly remember to follow project requirements by:

- Using a section tag for your projects and about me containers
- Please use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
- Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.
- You don't need to implement the detail modal window that requires JavaScript.
- Please kindly stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Please kindly implement the button interaction (enable, hover, etc.).

Screenshot from 2022-05-30 19-18-40

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @tinatsina,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * PR has a good summary description :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @tinatsina 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                <li>Home</li>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+    </div>
+    <!--- Top left/right image section -->
+    <div class="top-left-image"></div>
+    <div class="top-right-image"></div>
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply index.html]: Changed it to header tags

[the remark this answers is not in the document]
Comment (tinatsina) [reply index.html]: Changed to `nav` tag according to project requirements. Also updated to the suggested format.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                    Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                    If you like what you see and have a project you need coded, don’t hesitate and contact me.
+                </p>
+                <button class="colab" type="button">
+                    Start collaboration
+                </button>
+            </div>
+            <div class="sub-container-2"></div>
+        </div>
+    </section>
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply index.html:48]: Looked thoroughly at the Figma template and updated all missing or un-included errors.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <section>
+        <div class="home-container">
+            <div class="sub-container-1">
+                <h1 class="developer-name">
+                    Tinaye Tsinakwadi
+                </h1>
+                <p class="developer-story">
+                    Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                    If you like what you see and have a project you need coded, don’t hesitate and contact me.
+                </p>
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply index.html:41]: Added the font and typography to match exactly the template.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @tinatsina,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**